commons/util-el/src/main/java/io/github/microcks/util/el/function/RandomBooleanELFunction.java

```diff
@@ -25,8 +25,10 @@
  */
 public class RandomBooleanELFunction implements ELFunction {
 
+   private Random random = new Random();
```

Comment:
Just a side note: I think it would be even better to make this Random static or shared at an upper level like a `AbstractRandomELFunction`. This would allow us to circumvent this bug of not random generation mentioned by Sonar cloud but also to save more resources and share random objects between more instances.

Reply:
Yes of course I'm agree with you

Reply:
If you want I can add this on a new pr